content/source/docs/enterprise/private/vmware-setup-guide.html.md

```diff
@@ -92,14 +91,16 @@ the internal database or vault may result in serious performance issues.
 
 ### Object Storage (S3)
 
-An [S3 Standard](https://aws.amazon.com/s3/storage-classes/) bucket must be
+An [S3 Standard](https://aws.amazon.com/s3/storage-classes/) bucket, or compatible storage, must be
 specified during the PTFE installation for application data to be stored
 securely and redundantly away from the virtual servers running the PTFE
-application. This S3 bucket must be access via the network to the PTFE virtual
-machine as well as the postgres server. Vault is used to encrypt all
-application data stored in the S3 bucket. This allows for further [server-side
+application. This object storage must be accessible via the network to the PTFE virtual
+machine as well as the PostgreSQL server. Vault is used to encrypt all
```

Comment:
"as well as the PostgreSQL server": postgresql would never talk to the object store on its own.